src/sentry/integrations/vsts/integration.py

```diff
@@ -42,10 +44,12 @@ def get_project_config(self):
         disabled = False
         try:
             projects = client.get_projects(self.model.metadata['domain_name'])
-        except ApiError:
+        except Exception as e:
             # TODO(LB): Disable for now. Need to decide what to do with this in the future
             # should a message be shown to the user?
-            # Should we try refreshing the token? For VSTS that often clears up the problem
+            if INVALID_ACCESS_TOKEN == e.message:
```

Comment:
Is there a way to suggest to the user to reinstall their integration?

Reply:
Definitely don't have anything like that yet.